dataline-db/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
     api 'org.jooq:jooq-meta:3.13.4'
     api 'org.jooq:jooq:3.13.4'
     api 'org.postgresql:postgresql:42.2.16'
+    api 'org.testcontainers:postgresql:1.14.3'
```

Comment:
I disagree with that. We shouldn't have testcontainer be a dep for dataline-db. It should remain a testImplementation